python/chemiscope/structures/_ase.py

```diff
@@ -304,6 +305,66 @@ def _is_convertible_to_property(value):
                 return False
 
 
+def extract_vectors_from_ase(frames, key="forces", **kwargs):
+    """
+    Extract a vectorial atom property from a list of ase.Atoms
+    objects, and returns a list of arrow shapes. Besides the specific
+    parameters it also accepts the same parameters as
+    `array_from_vector`, which are used to define the style of the
+    arrows.
+
+    :param frames: list of ASE Atoms objects
+    :param key: name of the ASE atom property. Should contain
+       three components corresponding to x,y,z
+    """
+
+    vectors = []
+
+    for f in frames:
+        if key not in f.arrays:
+            raise IndexError(f"Key {key} not found in Atoms frame")
```

Comment:
```suggestion
            raise IndexError(f"Key {key} not found in `Atoms.arrays`")
```

this will avoid some confusion if people thing we are looking for stuff in `info`